sample-input/tiny-lattice/tiny-lattice.py

```diff
@@ -56,10 +53,13 @@
 
 log.py_printf('NORMAL', 'Creating cells...')
 
-cells = []
-cells.append(CellBasic(universe=1, material=uo2_id))
-cells.append(CellBasic(universe=1, material=water_id))
-cells.append(CellFill(universe=0, universe_fill=2))
+cells = list()
```

Comment:
I agree. I'll revamp the lattice cell indexing, and let you revamp this in your own PR.